en/configure-a-tidb-cluster.md

```diff
@@ -479,6 +479,22 @@ spec:
 >
 > If the TiKV version is earlier than 4.0.14 or 5.0.3, due to [a bug of TiKV](https://github.com/tikv/tikv/pull/10364), you need to configure the timeout `spec.tikv.evictLeaderTimeout` as large as possible to ensure that all Region leaders on the TiKV Pod can be evicted within the timeout. If you are not sure about the proper value, greater than '1500m' is recommended.
 
+### Configure graceful upgrade for TiCDC cluster
+
+During TiCDC upgrade, TiDB Operator evicts all Tables from TiCDC Pod before restarting TiCDC Pod. Only after the eviction is completed (which means the number of Tables on TiCDC Pod drops to 0) or the eviction exceeds the specified timeout (1500 minutes by default), TiCDC Pod is restarted. If TiCDC has fewer than 2 replicas, TiDB Operator forces an upgrade without waiting for the timeout.
```

Comment:
Do we need to capitalize the first character of `Pod`?

Reply:
> If TiCDC has fewer than 2 replicas

Do you mean we only have 1 TiCDC instance? If so, it's better to `If the TiCDC only 1 replica, ...`

Reply:
The term "replica" is from `TiCDCSpec.Replicas`. Not sure which term is better, what do you think @leoppro ?

Reply:
I think instance is better.

Reply:
> Do we need to capitalize the first character of `Pod`?

We use upper case in other places, see line 480.